controllers/metrics.go

```diff
@@ -46,12 +46,21 @@ var (
 			Buckets:   []float64{1, 10, 30, 60, 120, 180, 240},
 		},
 	)
+
+	clusterSummaryStatusGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "projectsveltos",
+			Name:      "cluster_summary_status",
+			Help:      "Status of ClusterSummary for managed clusters (1=Ready, 0=NotReady)",
```

Comment:
I am guessing here you want to check whether SveltosCluster are ready. In such case, the CRD is SveltosCluster and the repo where this needs to be tracked is sveltoscluster-manager controller (it has a reconciler for SveltosCluster instances)